### PR TITLE
fix: dependabot labelling

### PR DESCRIPTION
**Pull request by jmeridth**
- [x] add dependabot for bundler dependencies
- [x] update pull request template comments to bottom so we don't have to copy paste comment around comment when creating PR
- [x] fix CODE_OF_CONDUCT link in pull request template
- [x] drop CHANGELOG line in pull request template

Checklist:

* [ ] I have updated the necessary documentation
* [x] I have signed off all my commits as required by [DCO](../CONTRIBUTING.md#legal)
* [x] My build is green

### CODE REVIEW

**Review by njbbaer — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+Checklist:
+
+* [ ] I have updated the necessary documentation
+* [ ] I have updated the changelog, if necessary (CHANGELOG.md)
```
Comment (kkohrt) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: Should we have an "unreleased" section at the top of the changelog and ask people to specifically add their changes there? Given the lines below that say
> Maintainers will bump the version and do a release when they are ready to release (possibly multiple merged PRs). Please do not bump the version in your PRs.

If you don't have a version number or release date, making a changelog entry becomes a chicken and egg problem

**Review by kkohrt — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+* [ ] I have signed off all my commits as required by [DCO](../CONTRIBUTING.md#legal)
```
Comment (kkohrt) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Also, following the link above as rendered in this very PR, I find that the resulting URL:
https://github.com/procore-oss/blueprinter/CONTRIBUTING.md
returns a 404
The actual content is at:
https://github.com/procore-oss/blueprinter/blob/main/CONTRIBUTING.md

(thanks for finding that bug--I was planning to use relative URLs in another repo, and now I know what _won't_ work, lol)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jmeridth) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: https://github.com/procore-oss/blueprinter#releasing-a-new-version

I'm not a fan of the unreleased idea.  I get the perspective though.  I'm good with removing the CHANGELOG idea.  I'd prefer that be automated by GitHub actions.  Wdyt?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (jmeridth) [reply .github/PULL_REQUEST_TEMPLATE.md]: Fixed

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jmeridth) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed but willing to put back if you feel strongly about it

Comment (kkohrt) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed seems fine. 
We loose the narrative of the developer who made the change
But looking back at included pull requests should allow whomever cuts a release to fill things out.
Just as long as cutting a release includes updating the changelog :-)

**Review by kkohrt — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jmeridth) [reply .github/PULL_REQUEST_TEMPLATE.md]: I want to automate output like this

https://github.com/argoproj/argo-helm/releases/tag/argo-workflows-0.34.0

You see the list and the contributors

Comment (jmeridth) [reply .github/PULL_REQUEST_TEMPLATE.md]: Only possible really if we make [conventional git commits](https://www.conventionalcommits.org/en/v1.0.0/) required in the PR title.

**Review by njbbaer — APPROVED** [review]

**Review by kkohrt — APPROVED** [review]

**Pull request merged**